docs/docs/getting-started/lifecycle-management.md

```diff
@@ -266,10 +216,17 @@ An example of `KeptnAppContext`:
 This way, the `send-event` task is executed after the deployment of the whole application;
 in other words, after all of the workloads present in the `KeptnApp`
 are in a `Running` state.
+After the `send-event` task succeeds, `promotion` task will be executed.
 
 A detailed description of all the available fields of the `KeptnAppContext` resource can be found in the
 [KeptnAppContext API reference page](../reference/api-reference/lifecycle/v1beta1/index.md#keptnappcontext).
 
+> **Note**
+Please note, that to be able to execute promotion tasks, you need to have the promotion feature enabled.
+You can enable it via `lifecycleOperator.promotionTasksEnabled` helm value during installation of Keptn.
+More information can be found
+[here](https://github.com/keptn/lifecycle-toolkit/tree/main/lifecycle-operator/chart#global).
```

Comment:
I would prefer pointing to the published chart  not the developer ones

Reply:
we will publish this eventually and it could then be out of sync ... dunno

Reply:
hmm but we do not have it here yet https://github.com/keptn/lifecycle-toolkit-charts/tree/main/charts/keptn-lifecycle-operator, do we have some other stable place where we can point to?

Reply:
ok, adapted